internal/api/e2e/api/auth_test.go

```diff
@@ -375,7 +375,7 @@ func TestAuth_OAuthProvider_Token_WithLib(t *testing.T) {
 
 		authenticatedClient, err := config.NewRESTClient(config.RestConfigClient{
 			URL: persesBaseURL,
-			OAuth: &config.OAuth{
+			OAuth: &secret.OAuth{
```

Comment:
you need to import the `secret` package.

Reply:
Also, you will have to fix the usage of the struct `OAuth` that you moved from the `config` package to the `secret` package in the whole file